src/cluster.js

```diff
@@ -249,15 +98,214 @@ Cluster.getStorageType_ = function(type) {
 };
 
 /**
- * Set the cluster metadata.
+ * Create a cluster.
  *
- * See {@link Instance#createCluster} for a detailed explanation of
- * the arguments.
+ * @param {object} [options] See {@link Instance#createCluster}.
  *
- * @param {object} metadata Metadata object.
- * @param {string} metadata.location The cluster location.
- * @param {number} metadata.nodes Number of nodes allocated to the cluster.
- * @param {string} metadata.storage The cluster storage type.
+ * @example
+ * const Bigtable = require('@google-cloud/bigtable');
+ * const bigtable = new Bigtable();
+ * const instance = bigtable.instance('my-instance');
+ * const cluster = instance.cluster('my-cluster');
+ *
+ * cluster.create(function(err, cluster, operation, apiResponse) {
+ *   if (err) {
+ *     // Error handling omitted.
+ *   }
+ *
+ *   operation
+ *     .on('error', console.error)
+ *     .on('complete', function() {
+ *       // The cluster was created successfully.
+ *     });
+ * });
+ *
+ * //-
+ * // If the callback is omitted, we'll return a Promise.
+ * //-
+ * cluster.create().then(function(data) {
```

Comment:
This has a long history :) https://github.com/GoogleCloudPlatform/google-cloud-node/issues/2556

Reply:
Thanks for that context, I left a comment on a possible solution https://github.com/GoogleCloudPlatform/google-cloud-node/issues/2556#issuecomment-372682191

That should probably be addressed since (IMO) it would drastically improve the developer experience